### DISCUSSION TITLE
How to appropriately handle prod/dev environment variables

### DISCUSSION THREAD

**Discussion by powderham** [prisma/prisma#2310] [Q&A] [answered]
I have a nextjs project set up using prisma in the backend.

I largely followed the tutorial but am having difficulties setting the `DATABASE_URL` when trying to create a production build.

I currently have a `.env` file at `/prisma/.env` which gets read, but I can't seem to find a way to use a different env file e.g. `.env.prod` or `.env.local` when running `next build`.

Has anyone found a solution for this such that I can run `next build` or `next build --prod` and not have to worry about changing the `DATABASE_URL` manually.

Thanks.

**Comment by ryands17** [comment, marked as answer]
Hi @powderham :wave: 
One workaround would be to use sometihing like [dotenv-cli](https://www.npmjs.com/package/dotenv-cli#custom-env-files) and pass a custom `.env` file while running the command.

So it would look something like this:

```
dotenv -e env.prod -- next build --prod
```

Do let me know if it works.
There is also an open issue [here](https://github.com/prisma/prisma/issues/1717)

**Comment by powderham** [comment]
This indeed works, thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
